app/actions/websocket/teams.ts

```diff
@@ -49,3 +53,43 @@ export async function handleLeaveTeamEvent(serverUrl: string, msg: any) {
         }
     }
 }
+
+export async function handleUpdateTeamEvent(serverUrl: string, msg: WebSocketMessage) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
+    }
+
+    const teamFromData = JSON.parse(msg.data.team);
+    const team: Team = {
+        ...teamFromData,
+    };
+
+    database.operator.handleTeam({
+        teams: [team],
+        prepareRecordsOnly: false,
+    });
+}
```

Comment:
JSON.parse can throw

```suggestion
   try {
       const team = JSON.parse(msg.data.team) as Team;
       await database.operator.handleTeam({
            teams: [team],
            prepareRecordsOnly: false,
    });
   } catch {
       return;
   }
```